resources/home/dnanexus/utils/io.py

```
@@ -37,16 +46,46 @@
+
+    # check if we're filtering down the reference tracks by those provided
+    if select_tracks:
+        reference_urls["tracks"] = filter_reference_tracks(
+            select_tracks=select_tracks,
+            reference_tracks=reference_urls["tracks"],
+        )
+
+    template = merge(template, reference_urls, strategy=Strategy.ADDITIVE)
+
+    template["tracks"] = check_session_track_order(
+        session_tracks=template["tracks"]
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)